src/extract-cli.spec.js

```diff
@@ -0,0 +1,13 @@
+const fixtures = require('./test-fixtures.js');
+const fs = require('fs');
+const util = require('util');
+
+const exec = util.promisify(require('child_process').exec);
+
+describe('Command Line Usage', () => {
+  it('should output a correct POT file for vue component with $gettext used in template and data', async() => {
+    fs.writeFileSync('/tmp/TestComponent1.vue', fixtures.VUE_COMPONENT_WITH_GETTEXT_IN_TEXT_AND_DATA);
+    const {stdout} = await exec('node ./src/extract-cli.js --attribute v-translate /tmp/TestComponent1.vue ');
```

Comment:
nitpick: extra space at the end of your string. And maybe use a constant for the filename ?